### DISCUSSION TITLE
.Net 7.0 Windows Form InDesignHelpMode Not Working

### DISCUSSION THREAD

**Discussion by DavidLanham** [Krypton-Suite/Standard-Toolkit#975] [Q&A] [answered]
So I started a new project where I will be using .Net 7.0 and Windows Form.  When I add a Ribbon Control to my form and then try and use the InDesignHelpMode it is not working.  Does not allow me to click on the Tab to add a new tab or click on the Group to add a new group.  the Property for Ribbon InDesignHelpMode is set to True.

I started new project using ,Net 4.8 Framework and did the same steps and it works fine.

Am I doing something wrong?

**Comment by PWagner1** [comment, marked as answer]
Hi @DavidLanham 

No, it's not you. All the custom designers have been broken since .NET Core, see this [thread](https://github.com/Krypton-Suite/Standard-Toolkit/discussions/684) for more details.
